rpc.if

```diff
@@ -132,6 +132,43 @@ interface(`rpc_write_exports',`
 	allow $1 exports_t:file write_file_perms;
 ')
 
+########################################
+## <summary>
+##	Allow adding/removing exports files
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`rpc_add_rm_exports',`
+	gen_require(`
+		type exports_t;
+	')
+
+	allow $1 exports_t:file { create unlink };
+')
+
+
+########################################
+## <summary>
+##	Allow export directory operations
```

Comment:
The summary can be more descriptive: which operations?

Reply:
Interface removed